azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/b2c/UserPrincipal.java

```diff
@@ -127,7 +124,7 @@ public UserPrincipal(@NonNull Pair<JWSObject, JWTClaimsSet> jwtToken, String cod
         this.expireDate = jwtClaimsSet.getExpirationTime();
         this.notValidBeforeDate = jwtClaimsSet.getNotBeforeTime();
         this.issueAtDate = jwtClaimsSet.getIssueTime();
-        this.claim = jwtClaimsSet.getClaims();
```

Comment:
Do we need a test case?

Reply:
Yes, I will cover the code for UT later in other PRs.